packages/manager/modules/bm-server-components/src/general-information/installation/gabarit/server-installation-gabarit.controller.js

```diff
@@ -0,0 +1,511 @@
+                {
+                  t0: this.$scope.installation.server.nbPhysicalDisk,
+                  t1: this.$scope.informations.hardwareRaid.disks.length,
+                },
+              );
+              this.$scope.informations.hardwareRaidCompatible = false;
+            }
+          }
+        }
+      })
+      .catch((error) => {
+        this.$scope.resetAction();
+        this.Alerter.alertFromSWS(
+          this.$translate.instant(
+            'server_configuration_installation_ovh_fail_partition_schemes',
+            { t0: this.$scope.installation.server.name },
+          ),
+          error,
+          'server_dashboard_alert',
+        );
+      })
+      .finally(() => {
+        this.$scope.loader.loading = false;
+      });
+  }
+
+  deleteGabarit() {
+    this.atInternet.trackClick({
+      name:
+        'dedicated::dedicated::server::system-install::existing-template::delete-template::confirm',
+      type: 'action',
+    });
+    this.$scope.loader.loading = true;
+
+    this.Server.deleteGabarit(
+      this.$stateParams.productId,
+      this.$scope.installation.deleteGabarit.id,
+    ).then(
+      () => {
+        this.$scope.errorGab.ws = null;
+        this.$scope.installation.deleteGabarit = null;
+      },
+      (data) => {
+        this.$scope.errorGab.ws = this.$translate.instant(
+          'server_configuration_installation_gabarit_step1_delete_fail',
+          {
+            t0: this.$scope.installation.deleteGabarit.displayName,
+            t1: data.data.message,
+          },
+        );
+      },
+    );
+  }
+
+  // ------INSTALL------
+
+  startInstall() {
+    this.atInternet.trackClick({
+      name:
+        'dedicated::dedicated::server::system-install::existing-template::install',
+      type: 'action',
+    });
+
+    this.$scope.loader.loading = true;
+    this.Server.startInstallation(
+      this.$stateParams.productId,
+      this.$scope.installation.selectGabarit.id,
+      this.$scope.installation.selectPartitionScheme,
+      {
+        language: camelCase(this.$scope.installation.selectLanguage),
+        customHostname: this.$scope.installation.options.customHostname,
+        postInstallationScriptLink: this.$scope.installation.options
+          .postInstallationScriptLink,
+        postInstallationScriptReturn: this.$scope.installation.options
+          .postInstallationScriptReturn,
+        sshKeyName: this.$scope.installation.options.sshKeyName,
+        useSpla: this.$scope.installation.options.useSpla,
+        softRaidDevices:
+          this.$scope.informations.nbDisk > 2 &&
+          this.$scope.installation.nbDiskUse > 1
+            ? this.$scope.installation.nbDiskUse
+            : null,
+        noRaid:
+          this.$scope.installation.nbDiskUse === 1 &&
+          !this.$scope.informations.raidController,
+      },
+    )
+      .then(
+        (task) => {
+          set(task, 'id', task.taskId);
```

Comment:
If possible replace with task.id = task.taskId;

Reply:
We can't change because:
Assignment to property of function parameter